### DISCUSSION TITLE
how to config kafka consume group for kafka connector

### DISCUSSION THREAD

**Discussion by njalan** [trinodb/trino#21298] [Q&A]
In my case each topic need to apply the consume group access,So how to config the consume group ?
I got below error message and I think it is due to consume group missing.

2024-03-27T21:34:04.855+0800    WARN    Query-20240327_133327_00002_43y4u-376   org.apache.kafka.clients.NetworkClient  [Consumer clientId=consumer-1, groupId=null] Bootstrap broker xxxx:9093 (id: -1 rack: null) disconnected

**Comment by hashhar** [comment]
Trino Kafka connector doesn't use consumer groups since each worker needs to read pre-determined portion of the topic.

Why do you want to use consumer groups?
The error message has nothing to do with consumer groups. It's also just a warning and it's fine for KafkaClient to disconnect since it can and will reconnect to a different broker over time.
